tests/test_check_files_checksums_logging.py

```diff
@@ -49,8 +50,7 @@ def should_log_missing_item(self, subtests: SubTests) -> None:
 
         # When/Then
         with patch.object(self.logger, "error") as logger_mock:
-            with subtests.test(msg="Return code"):
-                assert main() == 1
-
+            with subtests.test(msg="Return code"), raises(DoesNotExist):
```

Comment:
You can join this line with the previous one, running all three context managers in one.